prometheus_client/exposition.py

```diff
@@ -140,6 +140,13 @@ def log_message(self, format, *args):
         """Log nothing."""
 
 
+class _PrintHandler(WSGIRequestHandler):
```

Comment:
I think we could just use WSGIRequestHandler instead of creating a new handler. It will log by default.